lib/lhm/printer.rb

```diff
@@ -41,11 +40,6 @@ def exception(e)
       private
 
       def write(message)
-        if (extra = @max_length - message.length) < 0
-          @max_length = message.length
-          extra = 0
-        end
-
```

Comment:
Why was this removed?

Reply:
Originally I removed this since `max_length` is not used anywhere in the code except in the unit tests, so I decided to omit this out. 

But I have added it back, since we don't want to change the existing structure of `Printer`.